include/swift/Sema/CSFix.h

```diff
@@ -2693,6 +2720,57 @@ class IgnoreResultBuilderWithReturnStmts final
   }
 };
 
+class IgnoreInvalidASTNode : public ConstraintFix {
```

Comment:
nit: Should this be final as well?
```suggestion
class IgnoreInvalidASTNode final : public ConstraintFix {
```

Reply:
Good catch!